tools/depends/target/python3/darwin_embedded.patch

```diff
@@ -255,7 +256,7 @@
      /* If we get here it's definitely an error */
  
      free_string_array(argvlist, argc);
-@@ -5116,6 +5136,12 @@
+@@ -5113,6 +5133,12 @@
```

Comment:
Is the current range information incorrect?

Reply:
Yes